src/warehouse/index.js

```diff
@@ -208,6 +208,20 @@ function setDataFromInputAndComputeColumnTypes(
   level = 0,
 ) {
   if (!input || !isObject(input)) return;
+  if (completePrefix.endsWith('context_traits_') && isStringLikeObject(input)) {
+    if (prefix === 'context_traits_') {
```

Comment:
Here we only handle `context_traits_`, but I was under the impression we need to handle `traits_` as well. (for group calls)

Reply:
Added support for traits in group calls